FreeRTOS-Plus/Source/FreeRTOS-IoT-Libraries-LTS-Beta2/c_sdk/platform/freertos/transport/include/sockets_freertos.h

```diff
@@ -19,9 +19,18 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
```

Comment:
I think it would be better to rename these files to `freertos_sockets_wrapper.c/h` to avoid confusion with the FreeRTOS+TCP files...

Reply:
Yea agree.